apps/channels/models.py

```diff
@@ -166,6 +172,10 @@ def webhook_url(self) -> str:
             is_secure=True,
         )
 
+    def soft_delete(self):
+        self.active = False
```

Comment:
It doesn't look like the `active` field is being used for anything else. We could rename it to `deleted` also if need be.

Reply:
Makes sense to rename it to `deleted` to avoid confusion